tests/apps/actorfeatures/app.go

```diff
@@ -0,0 +1,343 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+const (
+	appPort = 3000
+
+	actorMethodURLFormat = "http://localhost:3500/v1.0/actors/%s/%s/%s/%s"
+
+	registedActorType       = "testactorfeatures" // Actor type must be unique per test app.
+	actorIdleTimeout        = "1h"
+	actorScanInterval       = "30s"
+	drainOngoingCallTimeout = "30s"
+	drainBalancedActors     = true
+	secondsToWaitInMethod   = 5
+)
+
+type daprActor struct {
+	actorType string
+	id        string
+	value     int
+}
+
+// represents a response for the APIs in this app.
+type actorLogEntry struct {
+	Action         string `json:"action,omitempty"`
+	ActorType      string `json:"actorType,omitempty"`
+	ActorID        string `json:"actorId,omitempty"`
+	StartTimestamp int    `json:"startTimestamp,omitempty"`
+	EndTimestamp   int    `json:"endTimestamp,omitempty"`
+}
+
+type daprConfig struct {
+	Entities                []string `json:"entities,omitempty"`
+	ActorIdleTimeout        string   `json:"actorIdleTimeout,omitempty"`
+	ActorScanInterval       string   `json:"actorScanInterval,omitempty"`
+	DrainOngoingCallTimeout string   `json:"drainOngoingCallTimeout,omitempty"`
+	DrainBalancedActors     bool     `json:"drainBalancedActors,omitempty"`
+}
+
+var daprConfigResponse = daprConfig{
+	[]string{registedActorType},
+	actorIdleTimeout,
+	actorScanInterval,
+	drainOngoingCallTimeout,
+	drainBalancedActors,
+}
+
+type daprActorResponse struct {
+	Data     []byte            `json:"data"`
+	Metadata map[string]string `json:"metadata"`
+}
+
+// request for timer or reminder.
+type timerReminderRequest struct {
+	Data     string `json:"data,omitempty"`
+	DueTime  string `json:"dueTime,omitempty"`
+	Period   string `json:"period,omitempty"`
+	Callback string `json:"callback,omitempty"`
+}
+
+// requestResponse represents a request or response for the APIs in this app.
+type response struct {
+	ActorType string `json:"actorType,omitempty"`
+	ActorID   string `json:"actorId,omitempty"`
+	Method    string `json:"method,omitempty"`
+	StartTime int    `json:"start_time,omitempty"`
+	EndTime   int    `json:"end_time,omitempty"`
+	Message   string `json:"message,omitempty"`
+}
+
+var logs = []actorLogEntry{}
```

Comment:
Looks like it has not been changed... I saw that you renamed it in `actorapp`, but not in `actorfeatures` app

Reply:
Sorry, it was not intentional. Done.